src/helpers/i18nnext.js

```diff
@@ -45,6 +45,7 @@ i18n
 
     react: {
       wait: true, // globally set to wait for loaded translations in withTranslation hoc
+      useSuspense: true,
```

Comment:
Why is that good?
What that does?

Reply:
react-i18nnext uses Suspense by default, so this is more declaration to be sure we ARE doing it